### DISCUSSION TITLE
v0.75.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#6532] [Announcements]
## What's Changed

<img width="618" alt="image" src="https://github.com/outline/outline/assets/380914/ff00680a-9e8a-46a4-94fa-48622d0fcc3c">

This release brings a rebuild of the internal permissions in order to support inviting users to individual documents, this has been several months work and involved a complete redesign of the `Share` menu on documents. We have also worked to clarify public sharing and will be bringing similar enhancements to permission management in collections for the next release.

### Improvements

* Document title will now be automatically used when pasting internal doc url in https://github.com/outline/outline/pull/6352
* Embeds are now rendered within HTML and PDF exports in https://github.com/outline/outline/pull/6290
* Added ability to include drafts in search results by @apoorv-mishra in https://github.com/outline/outline/pull/6335
* Added 20+ new custom icons to use for collections
* Comments in the sidebar will now show the associated highlighted text in the document
* Installations with a single OIDC auth-provider will auto-redirect on login
* Retina images pasted into a document are now sized as they should be viewed in https://github.com/outline/outline/pull/6350
* Improved collection creation UI in https://github.com/outline/outline/pull/6485
* You can now drag a document or collection into the "Starred" section to star in the sidebar
* It's now possible to replace a file attachment inline
* Automatically replace (--) with emdash by @L11R in https://github.com/outline/outline/pull/6479
* Added native keyboard shortcuts for Find+Replace functionality
* Inline code now has a "Copy" action in the toolbar
* Added "Find and replace" option to menu on mobile in https://github.com/outline/outline/pull/6368
* Added a security setting to allow users to send invites in https://github.com/outline/outline/pull/6488
* Added [Camunda](https://camunda.com/) embed support in https://github.com/outline/outline/pull/6457
* Added support for [SmartSuite](https://app.smartsuite.com) embeds
* Added support for embedded images in Word documents exported from Confluence in https://github.com/outline/outline/pull/6361
* Added a "Copy as Markdown", and "Copy as Link" actions to documents
* Invite user screen now displays which collections they will be granted access to
* In-app notifications are now instant rather than delayed like emails in https://github.com/outline/outline/pull/6506
* API: Added missing `integrations.info` endpoint
* API: Added missing `comments.info` endpoint
* And lots more I've probably forgotten…

### Fixes

* Positioning of editing toolbar on mobile devices in https://github.com/outline/outline/pull/6279
* Suspended users now receive the correct error message on attempted login by @ruizehung in https://github.com/outline/outline/pull/6297
* Fixed an issue where Mermaid diagrams are not displayed correctly next to headings by @david50407 in https://github.com/outline/outline/pull/6299
* Fixed theme rendering of Mermaid diagrams by @david50407 in https://github.com/outline/outline/pull/6307
* Fixed Airtable embeds not working with latest url format
* Fixed an issue where Firefox would not behave correctly when resizing a video
* Fixed a server error when search term contains double single quotes
* Fixed an issue where pasted links from HTML could result in a double underline formatting
* Fixed a race condition that could result in corrupt structure in the collection sidebar
* Fixed an issue where settings could not be saved until an avatar was uploaded
* Fixed an issue that preventing unpublishing documents with archived children
* It is now possible to use parentheses before an @mention
* Fix flash on render of newly created comments https://github.com/outline/outline/pull/6322
* Database relationships are now cleaned up correctly when a user is deleted in https://github.com/outline/outline/pull/6343
* Add notifications for document and collection access in https://github.com/outline/outline/pull/6460
* Updates to user model are now synced between clients in https://github.com/outline/outline/pull/6490
* Improved resiliency of importer in https://github.com/outline/outline/pull/6519, https://github.com/outline/outline/issues/6273
* It is no longer possible to duplicate comment marks through copy/paste of text
* Pasting a markdown table will now always trigger markdown parsing

## New Contributors
* @ruizehung made their first contribution in https://github.com/outline/outline/pull/6297
* @david50407 made their first contribution in https://github.com/outline/outline/pull/6299
* @L11R made their first contribution in https://github.com/outline/outline/pull/6479

**Full Changelog**: https://github.com/outline/outline/compare/v0.74.0...v0.75.0

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.75.0'>v0.75.0</a>.</em>

**Comment by prologic** [comment]
Any plans to incorporate local LLM(s) such as Ollama to power the Questions and Answers feature for self-hosters? 🙏

**Reply by microbearlogist** [reply]
Great bunch of work!
Thank you very much for all these additions and corrections.

**Reply by klinshy** [reply]
That would be awesome!

**Comment by b-d-e** [comment]
This is great, thanks! Are there any plans to support sharing to _groups_ via this menu, rather than one user at a time?

**Reply by tommoor** [reply]
Yea, it's not top of the list but definitely to add this
